releasenotes/notes/36667.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
```

Comment:
I think there is already a release notes in the original fix? If so no need to repeat it for CNI since the config knob is the same.

Reply:
There is a release-notes_istio prow job for each PR, so the release notes is mandatory.

Reply:
You can add `release-notes-none` label.

Reply:
I added the label manually.

Reply:
Removed, thanks